web/src/recent_view_ui.js

```diff
@@ -975,9 +975,6 @@ export function complete_rerender() {
     // was not the first view loaded in the app.
     show_selected_filters();
 
-    // Update the banner now that it's rendered.
-    update_load_more_banner();
```

Comment:
Why did you remove this @amanagr ?

Reply:
Since we are calling it in `callback_after_render` now.